### PR TITLE
Add Healthcare Accelerator docs

### PR DESCRIPTION
**Pull request by isuruuy429**
This PR contains the following. 
- Overview of SMART on FHIR
- How to retrieve the Patient ID
- A few additions to manual.md

### CODE REVIEW

File: en/docs/install-and-setup/manual.md [hunk 1]
```diff
@@ -10,14 +10,37 @@
...
+    ```
+    ./wso2update_darwin
+    ```
+
+    Check the latest update level for the product. 
+    ```
+    ./wso2update_darwin check
+    ```
+
+    Update the product to the update level you need. As an example, if you need to update it to wso2am-4.2.0.118 we can provide the command as below.
```
Comment (sameeragunarathne) [inline en/docs/install-and-setup/manual.md]: Do we need to mention a product update level? Mentioning update using update tool to the latest is enough right?

File: en/docs/install-and-setup/manual.md [hunk 2]
```diff
@@ -10,14 +10,37 @@
...
+    ```
+
+    Update the current state of your product. 
+    ```
+    ./wso2update_darwin current-state
+    ```
+
+
+4. Extract WSO2 OH APIM Accelerator to `<WSO2_APIM_HOME>`. Let's call it `<WSO2_OH_APIM_ACC_HOME>`.
+5. [Optional] Check the accelerator configurations in <WSO2_HC_APIM_ACC_HOME>/conf/config.toml file to enable or disable features.
```
Comment (sameeragunarathne) [inline en/docs/install-and-setup/manual.md:37]: Let's have a note and table to discribe the accelerator config toml entries

File: en/docs/secure-health-apis/guides/getting-patientid.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+1. Navigate to Developer Portal in your web browser: `https://localhost:9443/devportal/`
+2. Self signup a user by clicking 'SIGN-IN' button, it will redirect you to the Authentication Portal. 
+3. Click on 'Create Account' link, provide a username and click on 'Proceed to Self Register'. 
+4. Fill in the details and create the account. 
+5. Log into Devportal with the crendetials of the above user. 
+6. Click on 'ADD NEW APPLICATION' button, provide application details and save the application. 
+7. Go to created application, and click on 'Production Keys' in left side panel. 
+8. For Grant Types, enable 'Code' grant type, provide the Callback URL. This is the app that we need to redirect. For the timebeing let's provide `http://localhost:3000`
+9. Click on 'GEENRATE KEYS', and click on 'UPDATE' button. 
+10. Note down the Consumer Key and Consumer Secret.
```
Comment (sameeragunarathne) [inline en/docs/secure-health-apis/guides/getting-patientid.md]: If we have already steps in the apim docs we should link to it and say to follow the steps. Also add screenshots where necessary

File: en/docs/secure-health-apis/guides/getting-patientid.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
+# Retrieving the Patient ID
```
Comment (sameeragunarathne) [inline en/docs/secure-health-apis/guides/getting-patientid.md]: Should be something like "Configuring SMART App Launch for a patient user"

File: en/docs/secure-health-apis/guides/getting-patientid.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+In this example we will walk through the steps that how we can get the Patient ID by executing the SMART on FHIR flow in Healthcare Accelerator.
```
Comment (sameeragunarathne) [inline en/docs/secure-health-apis/guides/getting-patientid.md]: Description needs to reflect that these steps walkthrough on configuring a SMART App Launch flow for a patient user.

Comment (isuruuy429) [reply en/docs/secure-health-apis/guides/getting-patientid.md]: Fixed in https://github.com/wso2/open-healthcare-docs/pull/44/commits/265eb2611a3f4cda6c4e230c05fb3f442d5afcee

File: en/docs/secure-health-apis/guides/getting-patientid.md (hunk 4 above)
Comment (isuruuy429) [reply en/docs/secure-health-apis/guides/getting-patientid.md]: Fixed in https://github.com/wso2/open-healthcare-docs/pull/44/commits/265eb2611a3f4cda6c4e230c05fb3f442d5afcee

File: en/docs/secure-health-apis/guides/getting-patientid.md (hunk 3 above)
Comment (isuruuy429) [reply en/docs/secure-health-apis/guides/getting-patientid.md]: Fixed in https://github.com/wso2/open-healthcare-docs/pull/44/commits/265eb2611a3f4cda6c4e230c05fb3f442d5afcee

File: en/docs/install-and-setup/manual.md (hunk 2 above)
Comment (isuruuy429) [reply en/docs/install-and-setup/manual.md:37]: Fixed in https://github.com/wso2/open-healthcare-docs/pull/44/commits/265eb2611a3f4cda6c4e230c05fb3f442d5afcee

File: en/docs/install-and-setup/manual.md (hunk 1 above)
Comment (isuruuy429) [reply en/docs/install-and-setup/manual.md]: Fixed in https://github.com/wso2/open-healthcare-docs/pull/44/commits/265eb2611a3f4cda6c4e230c05fb3f442d5afcee

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**